spec/index.bs

```diff
@@ -6691,7 +6726,337 @@ An <dfn dfn>Extent3D</dfn> is a {{GPUExtent3D}}.
+        <td>
+    <tr>
+        <td data-componenttype=int>{{GPUTextureFormat/rgba32uint}}
+        <td>&checkmark;
+        <td>&checkmark;
+        <td>
+    <tr>
+        <td data-componenttype=int>{{GPUTextureFormat/rgba32sint}}
+        <td>&checkmark;
+        <td>&checkmark;
+        <td>
+    <tr>
+        <td data-componenttype=float>{{GPUTextureFormat/rgba32float}}
+        <td>&checkmark;
+        <td>&checkmark;
+        <td>
+</table>
+
+### Depth/stencil formats ### {#depth-formats}
+
+All depth formats support {{GPUTextureUsage/COPY_SRC}}, {{GPUTextureUsage/COPY_DST}}, {{GPUTextureUsage/SAMPLED}}, and {{GPUTextureUsage/OUTPUT_ATTACHMENT}} usage. However, the source/destination is restricted based on the format.
+
+None of the depth formats can be filtered.
+
+<table class='data'>
+    <thead>
+        <tr>
+            <th>Format
+            <th>Bytes per texel
+            <th>Aspect
+            <th>Copy aspect from Buffer
+            <th>Copy aspect into Buffer
+    </thead>
+    <tr>
+        <td>{{GPUTextureFormat/stencil8}}
+        <td>1 &minus; 5
+        <td>stencil
+        <td colspan=2>&checkmark;
+    <tr>
+        <td>{{GPUTextureFormat/depth16unorm}}
+        <td>2
+        <td>depth
+        <td colspan=2>&checkmark;
+    <tr>
+        <td>{{GPUTextureFormat/depth24plus}}
+        <td>4
+        <td>depth
+        <td colspan=2>&cross;
+    <tr>
+        <td rowspan=2>{{GPUTextureFormat/depth24plus-stencil8}}
+        <td rowspan=2>4 &minus; 8
+        <td>depth
+        <td colspan=2>&cross;
+    <tr>
+        <td>stencil
+        <td colspan=2>&checkmark;
+    <tr>
+        <td>{{GPUTextureFormat/depth32float}}
+        <td>4
+        <td>depth
+        <td colspan=1>&cross;
+        <td colspan=1>&checkmark;
+</table>
+
+Copies between depth textures can only happen within the following sets of formats:
+  - {{GPUTextureFormat/stencil8}}, {{GPUTextureFormat/depth24plus-stencil8}} (stencil component), {{GPUTextureFormat/r8uint}}
+  - {{GPUTextureFormat/depth16unorm}}, {{GPUTextureFormat/r16uint}}
+  - {{GPUTextureFormat/depth24plus}}, {{GPUTextureFormat/depth24plus-stencil8}} (depth aspect)
+
+The {{GPUTextureFormat/depth32float}} format can only the destination if copied from another texture of the same format. It can also be a source of copies into {{GPUTextureFormat/r32float}}.
+
+Issue: clarify if `depth24plus-stencil8` is copyable into `depth24plus` in Metal.
+
+### Packed formats ### {#packed-formats}
+
+All packed texture formats support {{GPUTextureUsage/COPY_SRC}}, {{GPUTextureUsage/COPY_DST}}, and {{GPUTextureUsage/SAMPLED}} usages. All of these formats have {{GPUTextureComponentType/float}} type and can be filtered on sampling.
+
+<table class='data'>
+    <thead class=stickyheader>
+        <tr>
+            <th>Format
+            <th>Bytes per block
+            <th>Block Size
+            <th>Extension
+    </thead>
+    <tr>
+        <td data-componenttype=float>{{GPUTextureFormat/rgb9e5ufloat}}
```

Comment:
Does this still need to be in this section? If not, I think we should move it up and rename this section to "Compressed formats".

Reply:
I think it should be here, yes